controllers/databasecluster_controller.go

```diff
@@ -978,6 +971,38 @@ func (r *DatabaseClusterReconciler) genPXCBackupSpec(
 		}
 	}
 
+	if database.Spec.Backup.PITR.Enabled {
+		storageName := database.Spec.Backup.PITR.BackupStorageName
+		spec, backupStorage, err := r.genPXCStorageSpec(ctx, storageName, database.Namespace)
+		if err != nil {
+			return nil, errors.Join(err, errors.New("failed to get pitr storage"))
+		}
+		pxcBackupSpec.PITR.StorageName = pitrStorageName(storageName)
+
+		var timeBetweenUploads float64
+		if database.Spec.Backup.PITR.UploadIntervalSec != nil {
+			timeBetweenUploads = float64(*database.Spec.Backup.PITR.UploadIntervalSec)
+		}
+		pxcBackupSpec.PITR.TimeBetweenUploads = timeBetweenUploads
+
+		switch backupStorage.Spec.Type {
+		case everestv1alpha1.BackupStorageTypeS3:
```

Comment:
I know we don't support azure in the UI/API yet but since the operator already supports it for the rest of the backups maybe we can add it here as well.

Reply:
no, storage type other than s3 is not supported on the pxc operator's level, the [docs](https://docs.percona.com/percona-operator-for-mysql/pxc/backups-pitr.html):
> Both binlog and full backup should use s3-compatible storage to make point-in-time recovery work!